src/commonMain/kotlin/wizard/Dependencies.kt

```diff
@@ -308,3 +308,14 @@ val BuildKonfigPlugin = Dependency(
     catalogName = "buildKonfig",
     platforms = emptySet()
 )
+val MokoMvvm = Dependency(
```

Comment:
You are supposed to add it to the tests and check that all tests are green. (to `allDependencies` at least)